iot-hub/Samples/device/PnpDeviceSamples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -19,12 +19,20 @@ internal enum StatusCode
     {
         Completed = 200,
         InProgress = 202,
-        NotFound = 404,
-        BadRequest = 400
+        DeviceInitialProperty = 203,
+        BadRequest = 400,
+        NotFound = 404
     }
 
     public class TemperatureControllerSample
     {
+        // The default reported "value" and "av" for each "Thermostat" component on the client initial startup.
+        // See https://docs.microsoft.com/en-us/azure/iot-develop/concepts-convention#writable-properties for more details in acknowledgment responses.
```

Comment:
Remove `/en-us` part from URI so users get the language detected by their browser.

```suggestion
        // See https://docs.microsoft.com/azure/iot-develop/concepts-convention#writable-properties for more details in acknowledgment responses.
```